src/metis/cli/utils.py

```diff
@@ -117,8 +131,10 @@ def pretty_print_reviews(results, quiet=False):
                     )
                 if r.get("confidence") is not None:
                     print_console(
-                        f"    [magenta]Confidence:[/magenta] {r['confidence']}\n", quiet
+                        f"    [magenta]Confidence:[/magenta] {r['confidence']}", quiet
```

Comment:
Also needs escaping.

```suggestion
                        f"    [magenta]Confidence:[/magenta] {escape(r['confidence'])}", quiet
```
or if adopting the walrus operator
```suggestion
                        f"    [magenta]Confidence:[/magenta] {escape(confidence)}", quiet
```